com.jaquadro.NBTExplorer.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=NBTExplorer
+Comment=A Minecraft NBT file editor for editing world and player files.
```

Comment:
Comments should ideally not end with a dot. Also, it would be great if it was the same text as in AppStream metainfo summary.